dependency_licenses.csv

```diff
@@ -0,0 +1,53 @@
+cloud.google.com/go/compute/metadata,Unknown,Apache-2.0
```

Comment:
Do we need to store this file (in this format) ?

Reply:
If not, that's fine. We could remove the whole workflow around this. If you don't think it's necessary, then I can tear it all out.

I think it's good to have so that we can keep track of the dependency Licenses that PRs are adding.

Reply:
Makes sense :)

Let's keep it here.